src/main/java/com/codicesoftware/plugins/jenkins/tools/CmToolInstaller.java

```diff
@@ -16,6 +16,7 @@
 import java.io.IOException;
 import java.util.Collections;
 import java.util.List;
+import java.util.Objects;
```

Comment:
We can use `hudson.Util.fixEmpty()` instead

```suggestion
import hudson.Util;
```